source/tutorials/qiime2-for-experienced-microbiome-researchers.rst

```diff
@@ -0,0 +1,296 @@
+You can read more about the methods in the overview tutorial.
+
+DADA2 and deblur both output exact sequence variants, which supposedly represent the true biological sequences present in your data.
+Their creators have different terminology for these sequences (DADA2 calls them "amplicon sequence variants" (ASVs) and deblur calls them "sub-OTUs").
+We'll be using the ASV terminology throughout this tutorial to refer to both outputs.
+
+Preparing data for denoising
+''''''''''''''''''''''''''''
+
+Denoising requires little data preparation.
+Both DADA2 and deblur perform quality filtering, denoising, and chimera removal, so you shouldn't need to perform any quality screening prior to running them.
+That said, the official QIIME 2 tutorial recommends doing an initial :ref:`quality-filter <moving pictures deblur>` with default settings prior to using deblur (as recommended by the deblur developers).
+In our experience, DADA2 performs better without this step.
+
+Both methods have an option to truncate your reads to a constant length (which occurs prior to denoising).
+The truncating parameter is required for deblur and optional for DADA2.
+Reads shorter than the truncation length are discarded and reads longer are truncated at that position.
+The overview tutorial has more discussion on deciding what length you should truncate to.
+
+Denoising with DADA2
+''''''''''''''''''''
+
+The :doc:`DADA2 plugin </plugins/available/dada2/index>` has multiple methods to denoise reads:
+
+* :doc:`denoise paired-end <../plugins/available/dada2/denoise-paired/>` requires unmerged, paired-end reads (i.e. both forward and reverse).
+* :doc:`denoise single-end <../plugins/available/dada2/denoise-single/>` accepts either single-end or unmerged paired-end data. If you give it unmerged paired-end data, it will only use the forward reads (and do nothing with the reverse reads).
+* :doc:`denoise-pyro <../plugins/available/dada2/denoise-pyro/>` accepts ion torrent data.
+
+Note that DADA2 may be slow on very large datasets. You can increase the number of threads to use with the ``--p-n-threads`` parameter.
+
+Denoising with deblur
+''''''''''''''''''''''
+
+The :doc:`deblur <../plugins/available/deblur/index>` plugin has two methods to denoise sequences:
+
+* :doc:`denoise-16S <../plugins/available/deblur/denoise-16S/>` denoises 16S sequences.
+* :doc:`denoise-other <../plugins/available/deblur/denoise-other/>` denoises other types of sequences.
+
+If you use ``denoise-16S``, deblur performs an initial positive filtering step where it discards any reads which do not have a minimum 60% identity similarity to sequences from the 85% OTU GreenGenes database.
+If you don't want to do this step, use the ``denoise-other`` method.
+
+deblur can currently only denoise single-end reads.
+It will accept unmerged paired-end reads as input, it just won't do anything with the reverse reads.
+Note that deblur *can* take in *merged* reads and treat them as single-end reads.
```

Comment:
"so merge paired-end data prior to denoising with deblur."

is that kicking the horse?